cosima_cookbook/explore.py

```diff
@@ -941,3 +974,12 @@ def _load_variables(self):
         """
         self.var_selector.set_variables(self.variables)
         self.var_selector._filter_eventhandler(None)
+
+    def data(self):
+        """
+        Return xarray DataArray if one has been loaded
+        """
+        if self.loaded_data is None:
+            print('No data can be returned: no variable has been loaded')
```

Comment:
Go through `warnings`?

Reply:
I did, but it didn't seem to show up in the jupyter notebook, which is where I need it to be seen. Maybe I invoked it incorrectly?

Reply:
Hmm, I thought they just appeared normally without anything special. I don't think we suppress any warnings in the library either. Happy to leave it like this if at least it'll show up.

Reply:
I tried it again, and still no joy, `warnings.warn` didn't print to the notebook. Not sure if it is different that it is inside an `ipywidgets` class. There is some machinery to capture output and send it to a widget, but honestly I think I'll just leave the `print` statement there for the time being.